src/commands/config.ts

```diff
@@ -0,0 +1,9 @@
+import { Argv } from "yargs";
```

Comment:
Not sure to understand the purpose of this command ?

Reply:
PO requested for a command in the cli to be able to create config file for generic templates. When this command is issued, a config file would be that uses the with newly created wallet/docstore/token registry etc.

Reply:
please ping me when the MR is ready ?